frontend/src/summary.pug

```diff
@@ -91,15 +91,17 @@
   .fileTypes(v-if="filterBreakdown")
     .checkboxes.mui-form--inline(v-if="Object.keys(fileTypeColors).length > 0")
       label(
-        v-bind:style="{ 'background-color' : '#000000', 'color' : '#ffffff' }"
+        style='background-color: #000000; color : #ffffff'
```

Comment:
> minor style nit: maybe make it oneline here? since there's only one attribute
> There's an extra space before the : in color : #ffffff too

two more below for the `v-bind` as well